### ISSUE TITLE
The 'CustomiModelData starting value' configuration is invalid

### ISSUE THREAD

**Issue by yebanme** [PluginBugs/Issues-ItemsAdder#2753]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.

### What happened?

The ModeId assigned after modifying the value of CustomimModelData starting value in the configuration file is not the starting value of the configuration

### Steps to reproduce the issue

1.Modifying CustomimModelData starting value
`  items:
    CustomModelData-starting-value:
      ALL: 100000
      DIAMOND: 110000`
2.Edit files under this path contents\_iainternal\configs\icons.yml
3.Add the following content at the beginning of the file
`items:
  test1:
    enabled: true
    display_name: display-name-icon_left_blue
    resource:
      material: PAPER
      model_id: 10042
      generate: true
      textures:
        - icons/icon_left_blue`
4.Clear the contents of the storage folder and restart the server
5.In file items_ids_cache.yml, you will see the following content
`PAPER:
  _iainternal:test1: 10042
  _iainternal:icon_left_blue: 10043
  _iainternal:icon_right_blue: 10044
  _iainternal:icon_search: 10045
  _iainternal:icon_cancel: 10046
  _iainternal:icon_arrow_chest: 10047
  _iainternal:icon_back_orange: 10048
  _iainternal:icon_next_orange: 10049`
Among them, _iainternal:test1: 10042 is the modeId specified in the configuration, and the rest of the content did not generate a modeId according to the configuration requirements in the config

### Server version

This server is running Paper version git-Paper-31 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: a226f44)

### ItemsAdder Version

ItemsAdder version 3.5.0

### ProtocolLib Version

ProtocolLib version 5.1.0-SNAPSHOT-654

### LoneLibs Version

LoneLibs version 1.0.25

### Problematic items yml configuration file (optional)

```shell
items:
    CustomModelData-starting-value:
      ALL: 100000
      DIAMOND: 110000




info:
  namespace: _iainternal
items:
  test1:
    enabled: true
    display_name: display-name-icon_left_blue
    resource:
      material: PAPER
      model_id: 10042
      generate: true
      textures:
        - icons/icon_left_blue
  icon_left_blue:
    enabled: true
    display_name: display-name-icon_left_blue
    resource:
      material: PAPER
      generate: true
      textures:
        - icons/icon_left_blue
  icon_right_blue:
    enabled: true
    display_name: display-name-icon_right_blue
    resource:
      material: PAPER
      generate: true
      textures:
        - icons/icon_right_blue
```

### Other files, you can drag and drop them here to upload. (optional)

PAPER:
  _iainternal:test1: 10042
  _iainternal:icon_left_blue: 10043
  _iainternal:icon_right_blue: 10044
  _iainternal:icon_search: 10045
  _iainternal:icon_cancel: 10046
  _iainternal:icon_arrow_chest: 10047
  _iainternal:icon_back_orange: 10048
  _iainternal:icon_next_orange: 10049
POTION:
  __manually_handled:_iainternal__player_hat_slot: 100000
  __manually_handled:_iainternal__player_left_arm_slot: 100001
  __manually_handled:_iainternal__player_right_arm_slot: 100002
PLAYER_HEAD:
  __manually_handled:_iainternal__player_phead_0: 100000
  __manually_handled:_iainternal__player_parm_left_3: 100001
  __manually_handled:_iainternal__player_pbody_2: 100002
  __manually_handled:_iainternal__player_pleg_left_1: 100003
  __manually_handled:_iainternal__player_pleg_right_5: 100004
  __manually_handled:_iainternal__player_parm_right_4: 100005

**Comment by yebanme** [comment]
The information currently provided can replicate the problem that has occurred. I hope you can take the time to review this error, which affects the normal use of the plugin

**Comment by LoneDev6** [comment]
Will be checked and fixed, thanks.

Anyway forcing `model_id` is a feature just because some people asked for it, but it vanishes the whole purpose of ItemsAdder which is avoid worrying about CustomModelData and simply use the ItemsAdder commands or API to get items.

**Comment by yebanme** [comment]
> Will be checked and fixed, thanks.
> 
> Anyway forcing is a feature just because some people asked for it, but it vanishes the whole purpose of ItemsAdder which is avoid worrying about CustomModelData and simply use the ItemsAdder commands or API to get items.`model_id`

I agree with your opinion, but forcing the use of specific ModeID can better associate IA usage on some plugins. Thank you for your efforts

**Comment by yebanme** [comment]
Sorry, the issue has not been fixed in the latest ItemsAdder 3.5.0b, testing environment 
Paper git-Paper-43 (MC: 1.20.1)
ItemsAdder 3.5.0b
LoneLibs 1.0.25
ProtocolLib version 5.1.0-SNAPSHOT-654

**Comment by LoneDev6** [comment]
I tested exactly with the provided example in this first post message and it's working fine, it generated the CustomModelData IDs correctly.
You have to delete your cache file before, obviously.
